### ISSUE TITLE
Create tileset source throws error in v0.16

### ISSUE THREAD

**Issue by MatejSkrbis** [mapbox/mapbox-sdk-js#486]
I tried upgrading the library from v0.13.4 to v0.16.0

When calling this code to create a tileset source in v0.13.4 it works, but in v0.16.0 it throws an error:

```
const tileSourceResponse = await this.tilesets.createTilesetSource({
    id: sourceId,
    file: `./${id}.ldgeojson`
}).send();
```

Error is:
`Error creating tile source for 95c153b5-365d-4761-9b76-a5aa5b15d529: No file data in request. Expected 1 file named \"file\".`

**Comment by angelmunozs** [comment]
That's right, the same is happening to me and I think it's due to the changes introduced in version **[0.15.4](https://github.com/mapbox/mapbox-sdk-js/pull/479/files)**. The Mapbox API is responding with a 400 Bad Request.

I have downgraded to 0.15.3 for now until it's fixed...

**Comment by aditya-forerunner** [comment]
the offending PR is #479 , we had to downgrade as well
